docs/applications.md

```diff
@@ -175,6 +175,40 @@ An example `data` dictionary to specify on `POST /_matrix/client/r0/pushers/set`
 [APNs documentation]: https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CreatingtheNotificationPayload.html
 
 
+#### VoIP (voice calls and video calls) notifications on iOS
+
+iOS is capable of displaying an on-screen call notification (with answer/deny buttons)
+and playing a ringtone.
```

Comment:
```suggestion
and play a ringtone.
```

Reply:
'iOS is capable of ... play a ringtone' doesn't sound right to me. How are you parsing this sentence in such a way that this is right? I can't easily spot it :)

Reply:
Oh! I have no idea. Ignore me!